doc/changelog.rst

```diff
@@ -26,14 +26,14 @@ Description
 Detailed Notes
 
 - Implemented new structure of SmartSim documentation. Added examples
-  images and further detail of SmartSim components.
```

Comment:
Thanks for fixing the changelog.  Can you also fix the order in the detailed description section.  The last two items are out-of-order compared to the top short descriptions.

Reply:
fixed